docs/getting_started.md

```diff
@@ -0,0 +1,170 @@
+the Keycloak admin console ([localhost:8180/auth/admin](http://localhost:8180/auth/admin) or http://&lt;HOSTNAME&gt;:8080/auth/admin) and
+login with the admin user you created in the [add admin user](#add-admin) section.
+
+Start by creating a user role:
+
+* Select `Roles` from the menu
+* Click `Add Role`
+* Enter `user` as `Role Name`
+* Click `Save`
+
+Next create a user:
+
+* Select `Users` from the menu
+* Click `Add user`
+* Enter any values you want for the user
+* Click `Save`
+* Select `Credentials` from the tabs
+* Enter a password in `New Password` and `Password Confirmation`
+* Click on the toggle to disable `Temporary`
+* Click `Reset Password`
+* Click `Role Mappings`
+* Select `user` under `Available Roles` and click `Add selected`
+
+As an alternative to manually creating the role and user you can use the partial import feature in the admin console and import
+the file [config/partial-import.json](config/partial-import.json) into your realm.
+
+One more step, if you want to access the examples with the admin user you need to add the `user` role to admin user:
+
+* Select `Users` from the menu
+* Click `View all users`
+* Click `Edit` for admin user
+* Click `Role Mappings`
+* Select `user` under `Available Roles` and click `Add selected`
+
+## <a id="wildfly"></a>Start and Configure the <span>WildFly</span> Server
+
+Before starting the WildFly Server start by extracting the Keycloak client adapter into it.
+
+For WildFly 10 extract `keycloak-wildfly-adapter-${project.version}.zip` into WILDFLY\_HOME.
+
+If you plan to try the SAML examples you also need the SAML WildFly adapter. To do this for WildFly 10
+`keycloak-saml-wildfly-adapter-dist-${project.version}.zip` into WILDFLY\_HOME.
+
+The next step is to start WildFly server:
+
+1. Open a terminal and navigate to the root of the WildFly server directory.
+2. Use the following command to start the WildFly server:
+   ````
+   For Linux:   WILDFLY_HOME/bin/standalone.sh
+   For Windows: WILDFLY_HOME\bin\standalone.bat
+   ````
+3. To install the Keycloak adapter run the following commands:
+   ````
+   For Linux:
+
+     WILDFLY_HOME/bin/jboss-cli.sh -c --file=WILDFLY_HOME/bin/adapter-install.cli
+     WILDFLY_HOME/bin/jboss-cli.sh -c --command=:reload
+
+   For Windows:
+
+    WILDFLY_HOME\bin\jboss-cli.bat -c --file=WILDFLY_HOME\bin\adapter-install.cli
+    WILDFLY_HOME\bin\jboss-cli.bat -c --command=:reload
+   ````
+4. If you plan to try the SAML examples you also need to install Keycloak SAML adapter:
+
+   ````
+   For Linux:
+
+     WILDFLY_HOME/bin/jboss-cli.sh -c --file=WILDFLY_HOME/bin/adapter-install-saml.cli
+     WILDFLY_HOME/bin/jboss-cli.sh -c --command=:reload
+
+   For Windows:
+
+     WILDFLY_HOME\bin\jboss-cli.bat -c --file=WILDFLY_HOME\bin\adapter-install-saml.cli
+     WILDFLY_HOME\bin\jboss-cli.bat -c --command=:reload
+   ````
+
+# Examples
```

Comment:
All the links to examples are broken due to missing leading slash.

Reply:
Thank you, fixed.